### ISSUE TITLE
How to Search by partial mac address

### ISSUE THREAD

**Issue by darioajr** [justintaddei/arp-lookup#4]
I need locate all raspberry on my network, the prefix of mac addres is b8:27:eb, I need search the ips from all raspberry.

**Comment by justintaddei** [comment]
Great question! You could do something like this:

```typescript
const table = await arp.getTable();
const raspberries = table.filter(row => row.mac.startsWith('b8:27:eb'))
```

However, I feel like this is a pretty common use case so I went ahead and released v1.3!  
You can now do this:

```typescript
const raspberries = await arp.fromPrefix('b8:27:eb')
```

Let me know if this helps!  
  
---
### Also
If you know what port you're trying to connect to, you may want to give [@network-utils/tcp-scanner](https://www.npmjs.com/package/@network-utils/tcp-scanner) a try.

**Comment by darioajr** [comment]
Worked well, Thanks.
